src/main/java/org/onedatashare/transferservice/odstransferservice/service/InfluxCache.java

```diff
@@ -27,20 +27,20 @@ public class InfluxCache {
     @Autowired
     ThreadPoolManager threadPoolManager;
 
+    public enum ThroughputType {
+        READER,
```

Comment:
We most likely want to introduce the general throughput as well, which encompasses from the read to the write. Aka the complete process from reading to writing.

Reply:
Yeah I agree.